fox_scraper/fox_scraper.py

```diff
@@ -116,20 +131,34 @@ def verify_encoding(self, item_name):
         else:
             return item_name
 
-    def scrap_fox_catalog(self, soup_sub_categories, main_cat_num):
+    def get_full_catalog(self, soup_sub_categories, main_cat_num):
```

Comment:
the name suggest that it will return all catalogue not just for a specific category